docker/package/tezos_setup_wizard.py

```diff
@@ -143,15 +142,15 @@ def get_snapshot_mode_query(modes):
         prompt="The Tezos node can take a significant time to bootstrap from scratch.\n"
         "Bootstrapping from a snapshot is suggested instead.\n"
         "How would you like to proceed?",
-        help="A fully-synced local tezos-node is required for running a baking instance.\n"
-        "By default, service with tezos-node will start to bootstrap from scratch,\n"
+        help="A fully-synced local octez-node is required for running a baking instance.\n"
+        "By default, service with octez-node will start to bootstrap from scratch,\n"
```

Comment:
see 2 comments above for motivation:
```suggestion
        help="A fully-synced local Tezos node is required for running a baking instance.\n"
        "By default, the Tezos node service will start to bootstrap from scratch,\n"
```